tests/FSharp.Test.Utilities/Assert.fs

```diff
@@ -18,9 +18,18 @@ module Assert =
     let inline shouldContain (needle : string) (haystack : string) =
         haystack.Should().Contain(needle) |> ignore    
 
+    // One fine day, all of the 3 things below should be purged and replaced with pure Assert.Equal.
```

Comment:
That's basically the only thing happening in this PR. I am just putting all this functions in one place, not replacing anything.